### PR TITLE
Fixed everything with flake8 and fix error (player.py)

### PR DESCRIPTION
**Pull request by kutuzovrusss**
## Summary

Fix error (player.py)
Fixed everything with flake8

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by H4ckerxx44 — APPROVED** [review]
LGTM

**Review by Sheepposu — APPROVED** [review]
Looks good 👍

**Comment by ooliver1** [comment]
Just wondering, many of those commits just add an extra line between functions and an extra space in comments

File: nextcord/enums.py [hunk 1]
```diff
@@ -313,100 +314,101 @@ class AuditLogActionCategory(Enum):
...
-    stage_instance_create    = 83
-    stage_instance_update    = 84
-    stage_instance_delete    = 85
-    sticker_create           = 90
-    sticker_update           = 91
-    sticker_delete           = 92
-    thread_create            = 110
-    thread_update            = 111
-    thread_delete            = 112
+    guild_update = 1
```
Comment (TAG-Epic) [inline nextcord/enums.py:317]: This should not be formatted.

File: nextcord/enums.py [hunk 2]
```diff
@@ -313,100 +314,101 @@ class AuditLogActionCategory(Enum):
...
-            AuditLogAction.message_delete:        AuditLogActionCategory.delete,
-            AuditLogAction.message_bulk_delete:   AuditLogActionCategory.delete,
-            AuditLogAction.message_pin:           None,
-            AuditLogAction.message_unpin:         None,
-            AuditLogAction.integration_create:    AuditLogActionCategory.create,
-            AuditLogAction.integration_update:    AuditLogActionCategory.update,
-            AuditLogAction.integration_delete:    AuditLogActionCategory.delete,
+            AuditLogAction.guild_update: AuditLogActionCategory.update,
+            AuditLogAction.channel_create: AuditLogActionCategory.create,
+            AuditLogAction.channel_update: AuditLogActionCategory.update,
```
Comment (TAG-Epic) [inline nextcord/enums.py:370]: Same here

File: nextcord/gateway.py [hunk 3]
```diff
@@ -252,19 +261,19 @@ class DiscordWebSocket:
...
-    DISPATCH           = 0
-    HEARTBEAT          = 1
-    IDENTIFY           = 2
-    PRESENCE           = 3
-    VOICE_STATE        = 4
-    VOICE_PING         = 5
-    RESUME             = 6
-    RECONNECT          = 7
-    REQUEST_MEMBERS    = 8
+    DISPATCH = 0
```
Comment (TAG-Epic) [inline nextcord/gateway.py:264]: This should not be formatted

File: nextcord/opus.py [hunk 4]
```diff
@@ -62,39 +65,42 @@ class SignalCtl(TypedDict):
...
+# Some constants from opus_defines.h
 # Error codes
-OK      = 0
+OK = 0
 BAD_ARG = -1
 
 # Encoder CTLs
-APPLICATION_AUDIO    = 2049
-APPLICATION_VOIP     = 2048
+APPLICATION_AUDIO = 2049
```
Comment (TAG-Epic) [inline nextcord/opus.py:92]: This should not be formatted

File: nextcord/opus.py [hunk 5]
```diff
@@ -62,39 +65,42 @@ class SignalCtl(TypedDict):
...
+
 class DecoderStruct(ctypes.Structure):
     pass
 
+
 EncoderStructPtr = ctypes.POINTER(EncoderStruct)
 DecoderStructPtr = ctypes.POINTER(DecoderStruct)
 
-## Some constants from opus_defines.h
+# Some constants from opus_defines.h
```
Comment (TAG-Epic) [inline nextcord/opus.py:86]: Why was this changed?

File: nextcord/player.py [hunk 6]
```diff
@@ -151,7 +154,7 @@ def __init__(self, source: Union[str, io.BufferedIOBase], *, executable: str = '
 
         self._process: subprocess.Popen = self._spawn_process(args, **kwargs)
         self._stdout: IO[bytes] = self._process.stdout  # type: ignore
-        self._stdin: Optional[IO[Bytes]] = None
+        self._stdin: Optional[IO[bytes]] = None
```
Comment (TAG-Epic) [inline nextcord/player.py:157]: Why was this changed?

File: setup.py [hunk 7]
```diff
@@ -62,24 +62,25 @@
...
-      include_package_data=True,
-      install_requires=requirements,
-      extras_require=extras_require,
-      python_requires='>=3.8.0',
-      classifiers=[
+    },
+    version=version,
+    packages=packages,
+    license='MIT',
+    description='A Python wrapper for the Discord API forked from discord.py',
```
Comment (TAG-Epic) [inline setup.py:76]: What does this change? From 8 spaces to 4 or?

**Review by TAG-Epic — CHANGES_REQUESTED** [review]
A few things that should be reverted, although it looks pretty good. We should probably wait until the slash commands fork is finished with merging this so theres not so many merge conflicts.

File: nextcord/player.py (hunk 6 above)
Comment (ooliver1) [reply nextcord/player.py:157]: ^ this breaks the typehint

**Comment by ooliver1** [comment]
Ok back to this PR, could you revert those unnecessary formatting changes, the indent changes and removing "##" to "#" means it's not a title but a subheading which isn't intended

**Review by Cheesecake-dev — APPROVED** [review]

**Pull request closed without merging**